### ISSUE TITLE
Kubernetes/cluster-registry appears to be abandoned

### ISSUE THREAD

**Issue by chases2** [kubernetes/org#1499]
### Organization or repository
kubernetes/cluster-registry

### Describe the issue
The repository hasn't been modified since March 13th. Issues in the repository are currently rotting away. Consider archiving this repository?

**Comment by cblecker** [comment]
/assign @pmorie @quinton-hoole 
/sig multicluster
Assigning sig-multicluster chairs

**Comment by nikhita** [comment]
@pmorie @quinton-hoole friendly ping :)

**Comment by nikhita** [comment]
@pmorie @quinton-hoole Can you take a look at this?

**Comment by cblecker** [comment]
sent an e-mail to the sig list

**Comment by pmorie** [comment]
We discussed this in our May 26 meeting. The predominant sentiment seemed to be that the project is essentially defunct in its present form. It is possible that we will create something similar in the future, but the SIG does not want to be bound by prior decisions made in this repo. It is also likely that we will create any new repository for this in the `kubernetes-sigs` github org. I will send a message to the mailing list so that we can have a lazy consensus to archive this project.

**Comment by nikhita** [comment]
@pmorie :wave: Just following up. Has there been a date set for the lazy consensus already? Couldn't find one the mailing list.

**Comment by RainbowMango** [comment]
I don't think this repo should be archived, in addition, I've some idea about how this API should be enhanced. Please just wait for a while, we will discuss it at the SIG meeting.

**Comment by k82cn** [comment]
@pmorie , any conclusion there? we'd like to use this report as cluster-api for multi-cluster scenario.

**Comment by nikhita** [comment]
@k82cn @RainbowMango I'd recommend following up on the [mailing list](https://groups.google.com/g/kubernetes-sig-multicluster/c/4F4L8qpHuTA), github notifications can sometimes go to a black hole :upside_down_face:

**Comment by k82cn** [comment]
sure, thanks very much for your reminder :)

**Comment by pmorie** [comment]
Thanks for bumping, I see @RainbowMango's reply now.

If there's an objection, we should give this some additional discussion time in a SIG meeting. I have put an item on July 07, 2020 agenda to discuss. We will set a date for lazy consensus at that time.

**Comment by pmorie** [comment]
We have deferred a decision on whether the existing repository will be archived or not; currently collecting use cases for a future cluster registry concept in the community in a world-writable doc. More information [here](https://groups.google.com/g/kubernetes-sig-multicluster/c/qW4ZDBu4BKw). Anyone should feel welcome to add to this document; we'll discuss use-cases in coming SIG meetings. We'll provide notice when there is a 'last call' for use cases.

**Comment by nikhita** [comment]
/remove-lifecycle stale

@pmorie has a decision been taken already?

**Comment by mrbobbytables** [comment]
@pmorie do you know if a decision was reached? It's been some time now and the repo's still most recent activity is from 2+ years ago.

**Comment by nikhita** [comment]
Going ahead with the archival based on https://github.com/kubernetes/client-go/issues/914 and https://github.com/kubernetes/community/pull/5661

**Comment by nikhita** [comment]
Repo has been archived: https://github.com/kubernetes-retired/cluster-registry

Also sent a notice to the multicluster mailing list + k-dev - https://groups.google.com/g/kubernetes-dev/c/FaxRtx2t0nw

The following PRs should merge before this issue can be closed:
- https://github.com/kubernetes/org/pull/2602 - remove GitHub teams
- https://github.com/kubernetes/community/pull/5661 - remove repo from sigs.yaml

**Comment by nikhita** [comment]
All related PRs have merged, closing
